app/src/main/java/com/kylehanish/todo/interfaces/iTodoItemChangeListener.java

```diff
@@ -0,0 +1,20 @@
+package com.kylehanish.todo.interfaces;
+
+import android.view.View;
+
+import com.kylehanish.todo.classes.TodoItem;
+
+/**
+ * Created by Kyle Hanish on 4/29/18.
+ */
+
+public interface iTodoItemChangeListener {
+    void SaveTodoItem(TodoItem item, boolean isEdit);
+
+    void DeleteItem(int position, int itemID);
+
+    void EditItem(TodoItem item);
+
+    void PreviewItem(View view);
```

Comment:
Why pass views here and not TodoItem objects themselves?

Reply:
I originally did this for simplicity when working with the recycler view list item and didn't reanalyze how I implemented this.